src/__test__/redux/actions/samples/saveSamples.test.js

```diff
@@ -151,7 +151,7 @@ describe('saveSamples action', () => {
     try {
       await store.dispatch(saveSamples(mockprojectUuid, updatedSample1));
     } catch (e) {
-      expect(e).toEqual(errorMsg);
+      expect(e.statusCode).toEqual(400);
     }
```

Comment:
I think this should be
```
await expect(async() => {
    await store.dispatch(saveProject(mockProject.uuid, mockProject));
}).rejects.toThrow(endUserMessages.ERROR_SAVING)
```

Reply:
checking the status code is more reliable, and easier to change than error messages